yt/visualization/plot_modifications.py

```diff
@@ -914,13 +917,23 @@ def __init__(
         self.factor = _validate_factor_tuple(factor)
         self.clim = clim
         self.take_log = take_log
-        if plot_args is None:
-            plot_args = def_plot_args
-        self.plot_args = plot_args
+
+        _plot_args = def_plot_args.copy()
+        if plot_args is not None:
+            issue_deprecation_warning(
+                "`plot_args` is deprecated. "
+                "You can now pass arbitrary keyword arguments instead of a dictionary.",
+                since="4.3.0",
+                stacklevel=5,
+            )
+            _plot_args.update(plot_args)
+        _plot_args.update({"negative_linestyles": negative_linestyles, **kwargs})
+
+        self.plot_args = _plot_args
         self.label = label
         if text_args is None:
-            text_args = def_text_args
-        self.text_args = text_args
+            text_args = {}
+        self.text_args = {**def_text_args, **text_args}
```

Comment:
this fixes an unrelated bug where `def_text_args` is completely ignored if any text_arg is supplied.